src/app/shared/object-grid/search-result-grid-element/item-search-result/item/item-search-result-grid-element.component.html

```diff
@@ -18,6 +18,7 @@
       </span>
       <div class="card-body">
           <ds-type-badge *ngIf="showLabel" [object]="dso"></ds-type-badge>
+          <ds-access-status-badge *ngIf="showAccessStatus" [uuid]="dso.uuid"></ds-access-status-badge>
```

Comment:
Instead of repeated `*ngIf="showAccessStatus"` and its retrieval from config, can be moved to `ng-container` surrounding code in html of `AccessStatusBadgeComponent`